src/components/SlideOver/SlideOver.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react'
+import './SlideOver.css'
+import { Button, Icon, Space, Transition, Typography } from '../../index'
+
+interface Props {
+  className?: string
+  children?: React.ReactNode
+  title?: string
+  description?: string
+  visible: boolean
+  wide?: boolean
+  loading?: boolean
+  align?: 'right' | 'left'
+  alignFooter?: 'right' | 'left'
+  hideFooter?: boolean
+  customFooter?: React.ReactNode
+  onCancel?: any
+  onCancelText?: string
+  onConfirm?: any
+  onConfirmText?: string
+}
+
+const SlideOver = ({
```

Comment:
Renamed it to SlideOver, as this name probably better corresponds to what this component is for.

Reply:
Would you be ok if we rename it to SidePanel instead ? 

I think SlideOver and SideOver is causing some confusion.